packages/pluggableWidgets/gallery-text-filter-native/src/GalleryTextFilter.tsx

```diff
@@ -67,28 +67,30 @@ export function GalleryTextFilter(props: GalleryTextFilterProps<GalleryTextFilte
                     return <Text>{errorMessage}</Text>;
                 }
                 return (
-                    <FilterComponent
-                        name={props.name}
-                        delay={props.delay}
-                        styles={styles}
-                        placeholder={props.placeholder?.value}
-                        updateFilters={(value: string): void => {
-                            const attributeCurrentValue = props.valueAttribute?.value || "";
-                            if (value !== attributeCurrentValue) {
-                                props.valueAttribute?.setValue(value);
-                                props.onChange?.execute();
-                            }
-                            const conditions = attributes
-                                ?.map(attribute => getFilterCondition(attribute, value, props.defaultFilter))
-                                .filter((filter): filter is FilterCondition => filter !== undefined);
-                            filterDispatcher({
-                                getFilterCondition: () =>
-                                    conditions && conditions.length > 1 ? or(...conditions) : conditions?.[0],
-                                filterType: FilterType.STRING
-                            });
-                        }}
-                        value={props.defaultValue?.value}
-                    />
+                    <View style={styles.container}>
```

Comment:
why do we need an extra view here?
I can see that there is already a view wrapping the text input.